apps/system/test/unit/rocketbar_test.js

```diff
@@ -677,30 +677,45 @@ suite('system/Rocketbar', function() {
     assert.ok(hideResultsStub.calledOnce);
   });
 
-  test('handleInput()', function() {
-    MockService.currentApp = {
-      isPrivateBrowser: function() {
-        return false;
-      }
-    };
-    var showResultsStub = this.sinon.stub(subject, 'showResults');
-    var hideResultsStub = this.sinon.stub(subject, 'hideResults');
+  suite('handleInput()', function() {
 
-    // With input
-    subject.input.value = 'abc';
-    subject.results.classList.add('hidden');
-    subject.handleInput();
-    assert.ok(showResultsStub.calledOnce);
-    assert.ok(MockIACPort.mNumberOfMessages() == 1);
+    var showResultsStub, hideResultsStub;
 
-    // With no input
-    subject.input.value = '';
-    subject.results.classList.remove('hidden');
-    subject.handleInput();
-    assert.ok(hideResultsStub.calledOnce);
+    setup(function() {
+      MockService.currentApp = {
+        isPrivateBrowser: function() {
+          return false;
+        }
+      };
+      showResultsStub = this.sinon.stub(subject, 'showResults');
+      hideResultsStub = this.sinon.stub(subject, 'hideResults');
+      subject.active = true;
+    });
 
-    showResultsStub.restore();
-    hideResultsStub.restore();
+    test('With input', function() {
+      subject.input.value = 'abc';
+      subject.results.classList.add('hidden');
+      subject.handleInput();
+      assert.ok(showResultsStub.calledOnce);
+      assert.ok(MockIACPort.mNumberOfMessages() == 1);
+    });
+
+    test('With input, but inactive', function() {
+      subject.input.value = 'abc';
+      subject.active = false;
+      subject.results.classList.add('hidden');
+      subject.handleInput();
+      assert.ok(hideResultsStub.calledOnce);
+      subject.active = true;
+    });
+
+    test('With no input', function() {
+      // With no input
```

Comment:
nit: this comment is basically just a duplicate of the test description, so I would remove this.